pkgs/tools/networking/getmail6/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchFromGitHub, python3Packages }:
+
+python3Packages.buildPythonApplication rec {
+  pname = "getmail6";
+  version = "6.9";
+
+  src = fetchFromGitHub {
+    owner = "${pname}";
+    repo = "${pname}";
```

Comment:
```suggestion
    owner = pname;
    repo = pname;
```

Reply:
Thank you for your suggestion. I have implemented it.